### ISSUE TITLE
Bug: Runner displays wrong Step on failing test

### ISSUE THREAD

**Issue by ThomasLandauer** [Codeception/Codeception#6053]
Something is wrong with the display of the current "Step". I think this also happened in the past, but I don't know if it ever got fixed.

When I run an acceptance test file, and one test is failing and a subsequent test has an exception: In the red square "Test/Step/Fail" of the failing test, the current "Scenario Step" of the exception-throwing test is shown.

codeception/codeception 4.1.12

**Comment by Naktibalda** [comment]
@ThomasLandauer Please test if patch proposed in https://github.com/Codeception/Codeception/pull/5821 solves your issue.

**Comment by ThomasLandauer** [comment]
No :-(
I tried it with a Cest with 8 tests. The first fails, the second passes, the remaining 6 have errors.
I looked at a few things in `Console.php`:
* `$failedMetaStep` is empty string, so the patch doesn't do anything in fact.
* `count($this->failedStep)` is 6 on the first error-test, runs down to 1 on the last error-test, and is 0 on the fail-test. So to me this looks like the array runs out of elements, and therefore just repeats the last item.
